app/src/main/java/com/google/android/stardroid/search/PrefixStore.java

```diff
@@ -14,11 +14,7 @@
 
 package com.google.android.stardroid.search;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```

Comment:
Minor nit: can you expand the imports?

At some point I'll get around to publishing the coding standards for the app (it'll basically be modeled on Google's internal rules).